benchmark/opperf/utils/profiler_utils.py

```diff
@@ -248,12 +248,11 @@ def python_profile(func):
     @functools.wraps(func)
     def python_profile_it(*args, **kwargs):
         runs = args[1]
-        modified_args = (args[0], 1, args[2])
```

Comment:
I don't remember the reason why we need to write this way in the first place. @ChaiBapchya could you please review. @connorgoggins Have you run through existing tests to make sure this does not break any existing usage?

Reply:
@apeforest yes, ran full OpPerf suite with Python profiler with this change and everything passed.

Reply:
So the args that are passed to this function are
args[0] = op
args[1] = warmup / runs (number of times to run for warmup or number of times to run)
args[2] - rest of the args

The way it worked for native MXNet CPP profiler is that you could pass the runs
But for Python's time it function, we had to manually run for loop for the number of time to run.
So that's what I did there

we copy the number of runs in a variable in run and then run it that many number of times 
For each run, we use python time it function to time it and then take average, mean, max, etc values for each of those individual python time runs.

Makes sense? @apeforest

Reply:
@ChaiBapchya So basically you are saying we don't need to do this modified_args for python profiler, right? So @connorgoggins change is valid?

Reply:
We **do** need to modify the args to meet the requirement for capturing per run timing info using python's time_it function. This needs to handled in a way that doesn't break existing native profiler.